### PR TITLE
Removing unnecessary whitespace in content paragraphs

### LINKED ISSUE #20: Ensure proper encoding of free text fields
Having a proper encoding is important here as users have free text fields they can write into. As seen in https://github.com/SORSE/sorse.github.io/commit/a4031b2d229b03b802c0a11f12a5568a37161349 and https://github.com/SORSE/sorse.github.io/pull/248#issue-453969638 this can lead to issues with PDF rendering.

Here it is also important to differ between the different types of fields, e.g. the title is of especial importance for PDF rendering (see https://github.com/SORSE/sorse.github.io/pull/248#issuecomment-661684360).  So the change should also take care on this.

### LINKED ISSUE #27: Strip whitespace in contents
As mentioned by @Chilipp we should take care to properly strip whitespace in the paragraphs of the contribution contents.

```suggestion
The challenges facing research software development are manifold and have long been a major topic at RSE conferences.
```
remove whitespace at the end of paragraph

_Originally posted by @Chilipp in https://github.com/SORSE/sorse.github.io/pull/258#discussion_r460744374_

### PR DESCRIPTION
**Pull request by eileen-kuehn**
This PR ensures to remove unnecessary whitespace from content paragraphs.

Closes #27. Also closes #20.

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -59,7 +61,13 @@ def to_str(value: str, default: Optional[str] = None) -> Optional[str]:
 
 def to_text(value: str) -> Optional[str]:
     """Function ensures proper encoding of entities"""
-    return replace_text(value, TEXT_REPLACEMENTS)
+    if value:
+        value = replace_text(value, TEXT_REPLACEMENTS)
+        # also ensure proper line endings
+        for match in LINE_ENDINGS.finditer(value):
+            ending = match.group(0)
+            value = value.replace(ending, "\n\n")
```
Comment (maxfischer2781) [inline utils.py]: Consider to use [``re.sub``](https://docs.python.org/3/library/re.html#re.sub) to do this all at once.

**Review by maxfischer2781 — CHANGES_REQUESTED** [review]
Got a suggestion to do this builtin.

File: utils.py (hunk 1 above)
Comment (eileen-kuehn) [reply utils.py]: I updated the code and also included code from another PR (#30) as this could be similarly improved this way.

**Review by maxfischer2781 — APPROVED** [review]
Nice and crisp, go for it! 🎉

**Pull request merged**